build/parser.mjs

```diff
@@ -968,9 +977,8 @@ class Parser {
     if (target.EstimatedVaultedDate) {
       item.estimatedVaultDate = this.normalizeOggDate(target.EstimatedVaultedDate);
     }
-    if (target.Vaulted) {
-      item.vaulted = target.Vaulted;
-    }
+
+    item.vaulted = target.Vaulted || wikiaItem.vaulted;
```

Comment:
Just realized something, this shouldn't be a boolean or, because it's fine if it's false
```suggestion
    item.vaulted = typeof target.Vaulted === 'boolean' ? target.Vaulted : wikiaItem.vaulted;
```

Reply:
unless the ogg data is wrong or the wikiadata is wrong? that was I figured would happen in either case. if both are false then that's fine

Reply:
it's because in js undefined is cast to false in a boolean comparison, so if we have the target data and vault is false it'll ignore it and look for the wikia which could be false, true or undefined. If ogg is the default data source it doesn't make sense to ignore it when the vaulted state is false.

Reply:
Ah I understand now

Reply:
basically we take ogg data by default unless it's not a boolean (undefined) so if it's not we take whatever the wikia gives us undefined, false, true, in the previous scenario we would take true from ogg and if it's not true we would take undefined, true or false from wikia.

Reply:
Just remembered there's also nullish coalescing operator
```suggestion
    item.vaulted = target.Vaulted ?? wikiaItem.vaulted;
```

Reply:
this would be better, since `target.Vaulted` could be false, in which case you'd want to use that.